movies_test.py

```diff
@@ -0,0 +1,134 @@
+"""Tests the functions of the code.py file"""
+
+import movies
+
+def customer_db():
+    """Define Customer database for testing"""
+    return {
+        "Bob" : {"movies":[],
+                 "cash" : 100.0},
+        "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+                 "cash" : 10.0},
+        "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+                  "cash" : 120.0},
+        "Ricci": {"movies": [],
+                  "cash" : 4.0}
+    }
+
+def movie_db():
+    """Define movie database for testing"""
+    return {
+        "Xmen" : {"title" : "Xmen 8: The Xmennening", "seats_avaliable" : 10},
+        "Bromance" : {"title" : "The Bromance", "seats_avaliable" : 20},
+        "Gigli" : {"title" : "Gigli: The Play: The Book: The movie", "seats_avaliable" : 102}
+    }
+
+def test_enough_money():
+    """Tests the enough money function"""
+    customers = customer_db()
+    bob = customers["Bob"]
+    ricci = customers["Ricci"]
+    assert not movies.enough_money(ricci)
+    assert movies.enough_money(bob)
+
+def test_charge():
```

Comment:
Just reiterating, there should be an assertion in here to make sure you aren't sending people into debt.